migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MigrationAuthorizationManager.java

```diff
@@ -83,4 +83,9 @@ public void throwIfNotWritable(SecurableByProfile resource) throws IOException,
     return new ArrayList<>(unfiltered);
   }
 
+  @Override
+  public void throwIfNonAdminOrMatchingOwner(User owner) throws IOException, AuthorizationException {
+
+  }
```

Comment:
Yes, MigrationAuthorizationManager is used to bypass authorization and force use of a configured user when using the MISO Service layer outside of Spring. There is no reason to perform auth checks during migration, and the DefaultAuthorizationManager makes use of a security context, which doesn't exist in migration either